### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by maxbischof**
* Remove not needed styled components
* Change folder names to lower case
* Add key to characters cards
* Adds set headline to locationpage
* Wraps set headline in use effect because of the following warning. I cant really explain it. If somebody has an explanation feel free to share it here.
Bildschirmfoto 2020-08-20 um 08 56 50

### CODE REVIEW

**Review by mpagels — CHANGES_REQUESTED** [review]
Got an error:
./src/App.js
Cannot find file: 'EpisodesPage.js' does not match the corresponding name on disk: './src/components/pages/episodes/Episodes'.

did you change the directory names to lowerCase as well?

File: src/components/pages/Episodes/EpisodesPage.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ function EpisodesPage({ setHeadline }) {
   const [episodes, setEpisodes] = useState([])
   const [pages, setPages] = useState()
 
-  setHeadline('Episodes')
+  useEffect(() => {
+    setHeadline('Episodes')
+  })
```
Comment (mpagels) [inline src/components/pages/Episodes/EpisodesPage.js:13]: Without empty list as second argument, the function will run every render. Is this necessary?

File: src/components/pages/Locations/LocationPage.js [hunk 2]
```diff
@@ -3,10 +3,14 @@ import Location from './LocationCard'
...
 
-export default function LocationPage() {
+export default function LocationPage({ setHeadline }) {
   const [locations, setlocations] = useState([])
   const [pages, setPages] = useState()
 
+  useEffect(() => {
+    setHeadline('Locations')
+  })
+
```
Comment (mpagels) [inline src/components/pages/Locations/LocationPage.js:13]: same as EpisodePage.js

File: src/components/pages/characters/CharactersPage.js [hunk 3]
```diff
@@ -4,7 +4,9 @@ import CharactersCard from './CharactersCard'
 export default function CharactersPage({ setHeadline }) {
   const [character, setCharacter] = useState([])
 
-  setHeadline('Characters')
+  useEffect(() => {
+    setHeadline('Characters')
+  })
```
Comment (mpagels) [inline src/components/pages/characters/CharactersPage.js:9]: same as LocationPage.js as well ;-)

**Comment by maxbischof** [comment]
Directorynames have been changed: https://github.com/neuefische/hh-2020-w4-rickandmorty-team1/pull/16/commits/2ebb6c982651e54d68f50cc99eb332faf1583893

File: src/components/pages/Episodes/EpisodesPage.js (hunk 1 above)
Comment (maxbischof) [reply src/components/pages/Episodes/EpisodesPage.js:13]: Headline in Header should be set on every render. Shouldn't it?

Comment (maxbischof) [reply src/components/pages/Episodes/EpisodesPage.js:13]: As mentioned above wihtout the useeffect there was this warning message Bildschirmfoto 2020-08-20 um 08 56 50

**Review by annidrumm — COMMENTED** [review]
Thank you for cleaning up!
As Martin mentioned it is important that all connections are set to lowercase and everything is still working. It would be nice if we change the name from `LocationPage` to plural as well. (`LocationsPage`)
Great work! And lets discuss the render error in our meeting.

**Comment by maxbischof** [comment]
Directory name changes hav to be added with `git mv`. Problem was solved by https://github.com/neuefische/hh-2020-w4-rickandmorty-team1/pull/16/commits/15850a43f99d20fe8aa900eff3412d585a4f474a

**Pull request merged**